schema/Periodical.schema.yaml

```diff
@@ -0,0 +1,31 @@
+title: Periodical
+'@id': stencila:Periodical
+extends: CreativeWork
+role: secondary
+status: experimental
```

Comment:
I think this an the other types in this PR should be marked as `unstable`. From the docs...

- `experimental`: new types (i.e. not defined on schema.org or elsewhere) that are still under development
- `unstable`: types that are defined elsewhere (e.g. on http://bioschemas.org) but for which the schema is still being developed
- `stable`: types for which the schema definition can be considered stable